scripts/zones/Lower_Jeuno/npcs/Chululu.lua

```diff
@@ -1,115 +1,149 @@
 -----------------------------------
 -- Area: Lower Jeuno
 --  NPC: Chululu
--- Starts and Finishes Quests: Collect Tarut Cards, Rubbish Day
+-- Starts and Finishes Quests: Collect Tarut Cards, Rubbish Day, All in the Cards
 -- Optional Cutscene at end of Quest: Searching for the Right Words
 -- !pos -13 -6 -42 245
 -----------------------------------
-require("scripts/globals/settings");
-require("scripts/globals/titles");
-require("scripts/globals/keyitems");
-require("scripts/globals/shop");
-require("scripts/globals/quests");
-local ID = require("scripts/zones/Lower_Jeuno/IDs");
+require("scripts/globals/settings")
+require("scripts/globals/titles")
+require("scripts/globals/keyitems")
+require("scripts/globals/shop")
+require("scripts/globals/quests")
+local ID = require("scripts/zones/Lower_Jeuno/IDs")
 -----------------------------------
 
 function onTrade(player,npc,trade)
-    if (player:getQuestStatus(JEUNO,tpz.quest.id.jeuno.COLLECT_TARUT_CARDS) == QUEST_ACCEPTED) then
+    if player:getQuestStatus(JEUNO,tpz.quest.id.jeuno.COLLECT_TARUT_CARDS) == QUEST_ACCEPTED then
         if (trade:hasItemQty(558,1) == true and trade:hasItemQty(559,1) == true and trade:hasItemQty(561,1) == true and trade:hasItemQty(562,1) == true and trade:getItemCount() == 4) then
-            player:startEvent(200); -- Finish quest "Collect Tarut Cards"
+            player:startEvent(200) -- Finish quest "Collect Tarut Cards"
         end
     end
-end;
+    if player:getQuestStatus(JEUNO,tpz.quest.id.jeuno.ALL_IN_THE_CARDS) >= QUEST_ACCEPTED then
```

Comment:
Remove the "end" right before here and use `elseif` - the only reason to use two separate end/end blocks is if both things can execute, which here that is not the case.